packages/cli/src/lib/print-config/print-config-command.unit.test.ts

```diff
@@ -25,17 +31,39 @@ describe('print-config-command', () => {
       { ...DEFAULT_CLI_CONFIGURATION, commands: [yargsConfigCommandObject()] },
     ).parseAsync();
 
-    expect(console.info).not.toHaveBeenCalledWith(
-      expect.stringContaining('"$0":'),
+    const log = ui().logger.getLogs();
+    expect(log).toEqual(
+      expect.arrayContaining([
+        {
+          stream: 'stdout',
+          message: expect.not.stringContaining('"$0":'),
+        },
+      ]),
     );
-    expect(console.info).not.toHaveBeenCalledWith(
-      expect.stringContaining('"_":'),
+    expect(log).toEqual(
+      expect.arrayContaining([
+        {
+          stream: 'stdout',
+          message: expect.not.stringContaining('"_":'),
+        },
+      ]),
     );
-    expect(console.info).toHaveBeenCalledWith(
-      expect.stringContaining('"outputDir": "destinationDir"'),
+    expect(log).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({
+          stream: 'stdout',
+          message: expect.stringContaining('"outputDir": "destinationDir"'),
+        }),
+      ]),
```

Comment:
I currently see vast majority of cases covering methods that print to `stdout` (only one test in CLI currently asserts `console.error`).

Alternatively, the helper/setup of the assertion should imo at least get rid of the array and other properties of the object, so that we are just asserting equality for `{ stream: 'stdout', message: <some string>}` and nothing else.

Reply:
done